legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -51,7 +51,6 @@ def validate(filing_json: dict) -> Optional[Error]:  # pylint: disable=too-many-
         return msg  # Cannot continue validation without legal_type
 
     msg.extend(validate_business_in_colin(filing_json, filing_type))
-    msg.extend(validate_continuation_in_authorization(filing_json, filing_type))
```

Comment:
We still need to validate if a file exist or not against this key

Reply:
Okay so I moved other validations to file submission.
I will add a validation to check if file exists.